src/main/resources/config.yml

```diff
@@ -43,6 +43,15 @@ notifications:
   email:
     fromAddress: ${EMAIL_NOTIFICATION_FROM_ADDRESS:-noreply@zerobase.io}
 
+metrics:
+  frequency: 1 minute
+  reporters:
+    - type: datadog
+      host: TODO
+      transport:
+        type: http
+        apiKey: ${DATADOG_API_KEY} #TODO
```

Comment:
@jasonspriggs may have this- I just got us an open source plan that they'll upgrade to whatever their enterprise plan is, ideally with the same keys.

Reply:
Also hi ❤️